Backend/src/constants.json

```
@@ -0,0 +1,9 @@
+{
+    "CAR_PORT": 4003,
+    "CAR_SERVER": "localhost",
```

Bad naming convention on my end (what we were using previously was incorrect). It would be good to correct it now by differentiating between the test address (localhost) and the pi's address (192.168.1.15).
```suggestion
    "TEST_ADDRESS": "localhost",
```